rg_instructor_analytics/views.py

```diff
@@ -491,6 +495,121 @@ def process(self, request, **kwargs):
         return JsonResponse(data={'student_info': student_info, 'exam_names': exam_names})
 
 
+class CohortView(AccessMixin, View):
+    """
+    Api for cohort statistic.
+    """
+
+    @staticmethod
+    def generate_cohort_by_mean_and_dispersion(student_info):
+        """
+        Generate cohort.
+
+        Generate cohort for next algorithm:
+        1. Calculate mean(m) and standart deviation(s) of total grade [0;1]
```

Comment:
typo: s/[0;1]/[0:1]/

Reply:
According to mathematics notation I change to [0,1]